index.html

```diff
@@ -46,10 +46,10 @@
   </head>
   <body>
     <div class="card">
-      <img src="my-profile-picture.png" alt="Girlcoder" style="width: 100%" />
-      <h1>Zandile Doe</h1>
+      <img src="Boipelo.jpg" alt="BoipeloM" style="width: 100%" />
+      <h1>Boipelo Magano</h1>
       <p class="title">Software developer</p>
-      <p>Girlcode Bootcamp</p>
+      <p>GirlCoder</p>
```

Comment:
This should not be committed for the addition to the yearbook, only 2 files need changes:

- `2021/yearbook.html` 
- `2021/images/Boipelo.jpg`


You can revert your commit or put back the old content please.